aerleon/lib/nokiasrl.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2023 Nokia All Rights Reserved.
```

Comment:
Just want to check on this since you are contributing with your personal gmail account. You are wishing to provide copyright to Nokia and you can attest that the code made was created by you on behalf of Nokia?

You can also add the company to the `AUTHORS` file in the root directory if you desire.

Reply:
Yes, copyright belongs to Nokia and I created the code on its behalf

Added the company name to AUTHORS